go.mod

```diff
@@ -1,10 +1,10 @@
 module github.com/hashicorp/vault-plugin-auth-jwt
 
-go 1.20
+go 1.21
```

Comment:
Don't forget to bump .go-version as well.

Reply:
Good call!